README.md

```diff
@@ -123,8 +123,159 @@ If you're wondering what a specific protocol does, your friend Google is always
+- **Type**: IPv4 (0x0800)  
+_Indicates which upper layer protocol should be used_  
+
+##### Network Layer
+
+![](network.png)  
+_Screenshot of the network layer information we have about the DNS packet._  
+
+- **Total Length**: 72  
+_Length of the IP packet (20 bytes - the header, the rest - data)_  
+- **Identification**: 0x2807 (10247)  
+_A unique ID of the IP packet_  
+- **Flags**: 0x4000, Don't fragment  
+_Any flags set for the packet. In our case, "Don't fragment" bit is set to 1, which forbids packet fragmentation_  
+- **Time to live**: 64  
+_The maximum time (in seconds) the packet is allowed to exist on the network_  
+- **Protocol**: UDP (17)  
+_Indicates which upper layer protocol should be used. Number inside the parentheses indictes an assigned port number for that protocol on the IP layer_  
+- **Header checksum**: 0xfe18 \[validation disabled]  
+_The header checksum value and its validation status_  
+- **Source**: 10.0.0.132  
+_An IPv4 address of the system that sent the packet_
+- **Destination**: 10.0.0.2  
+_An IPv4 address of the system that received the packet_
+
+##### Transport Layer
+
+![](transport.png)  
+_Screenshot of the transport layer information we have about the DNS packet._  
+
+- **Source Port**: 35540  
+_Port from which the packet has been sent_  
+- **Destination Port**: 53  
+_Port to which the packet has been sent_  
+- **Length**: 52  
+_Length of the UDP packet (8 bytes - the header, the rest - data)_  
+- **Checksum**: 0x2d94 \[unverified]  
+_The checksum value and its verification status_  
+
+##### Application Layer
+
+![](dns.png)  
+_Screenshot of the application layer information we have about the DNS packet._  
+
+- **Transaction ID**: 0x332e  
+_A unique ID of the query-response pair_  
+- **Flags**: 0x0100 Standard query  
+_Any flags set to the packet. In our case, this packet is a query (0), a standard one (0), not truncated (0), recursively done (1), Z reserved (0), and doesn't accept non-authenticated data (0)_  
+- **Questions**: 1  
+_Number of requests_  
+- **Answer RRs**: 0  
+_Number of answers_  
+- **Queries**: www.offensive-security.com: type A, class IN  
+_All requests included in this packet. In our case, we only have one_  
+
+As we can see, there is plenty of information about the packets that we can access with Wireshark. All the things we were able to tell about the DNS query packet #3 is visible by just clicking on the packet in the _Packet Listing_ window and paying close attention to the fields. However, this is a single packet. What if we want to search through hundreds of packets to find something specific? This is where our next topic comes in - sorting and filtering.
+
+### Sorting and Filtering
+
+#### Sorting Columns
+
+Let's look back at where we started: the _Packet Listing_ window. We saw all the standard columns like **Number** or **Protocol**. But what if we want to sort captured packets by something else? For example the **Destination Port**? 
```

Comment:
I think it should be "For example, the Destination Port" (Comma after for example)